### ISSUE TITLE
Issue and Pull Request Templates for Enhanced Collaboration

### ISSUE THREAD

**Issue by UTSAVS26** [qtop/qtop#331]
### Description:
As a participant in Hacktoberfest, I am excited to improve the collaboration process in our repository by adding structured issue and pull request templates. These templates will help contributors clearly communicate their ideas, report bugs, or propose new features, ensuring that all necessary information is provided for effective review and resolution.

#### Key Features:
- **Issue Template**: This will guide users in reporting bugs or suggesting enhancements, prompting them to include essential details like steps to reproduce, expected vs. actual behavior, and any relevant screenshots.
  
- **Pull Request Template**: This template will streamline the process for contributors submitting code changes, encouraging them to describe their modifications, reference related issues, and outline any necessary context for reviewers.

By implementing these templates, we aim to foster a more organized and efficient development workflow, making it easier for all contributors to participate and collaborate effectively.

**Comment by UTSAVS26** [comment]
@fgeorgatos  please assign this issue to me.

**Comment by fgeorgatos** [comment]
I'm with the impression this is a duplication of this issue: https://github.com/Vinit710/MEDAI/issues/23 and https://github.com/Vinit710/MEDAI/pull/25 ?

**Comment by UTSAVS26** [comment]
> I'm with the impression this is a duplication of this issue: https://github.com/Vinit710/MEDAI/issues/23 and https://github.com/Vinit710/MEDAI/pull/25 ?

Yes, it's somewhat a duplication, but not exactly the same. The new work will bring improvements, handle additional scenarios, and make things more efficient compared to the earlier issue. It also includes updates to align with the current version of the project.

**Comment by fgeorgatos** [comment]
treated in #411
